src/api/routes/offer.js

```diff
@@ -25,7 +26,7 @@ module.exports = (app) => {
     /**
      * Creates a new Offer
      */
-    router.post("/", authMiddleware.isGod, validators.create, async (req, res, next) => {
```

Comment:
This PR is incomplete unless you add the `companyMiddleware.isCompanyRep` before the `companyMiddleware.canCreateOffer`